app/src/main/java/org/projectbuendia/client/utils/Utils.java

```diff
@@ -50,7 +52,9 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-/** Utility methods. */
+/**
```

Comment:
Could you undo this reformat?  Comments that can fit on one line should stay on one line.

Reply:
Actually I'm seeing now that your IDE must have reformatted most of this file.  Please revert the reformatting-only changes because they make it harder to see what you meant to change.  It looks like you only meant to add the one static method `sortObsRows`, I think?

For future reference, the [style conventions in this codebase](https://github.com/projectbuendia/buendia/wiki/Java-Style) generally prioritize efficient use of screen space to counterbalance Java's verbosity.  For example, we don't use line breaks after `@Override`, and comments that fit on one line should be kept on one line.

Reply:
Pardon the auto-format stomping. Is there a code styles file to import?